README.md

```diff
@@ -4,14 +4,16 @@ Python implementation of the [STAR Voting](https://www.starvoting.org/) system.
 ## Getting started
 Install the required packages
 
-`pip install -r requirements.txt`
+```shell
+pip install .
+```
 
 ## Usage
 ### Single winner STAR
 To run a single winner election create a pandas dataframe of the ballot data and pass it into the STAR function
 
-```
-from STAR import STAR
+```python
+from starpy.STAR import STAR
 import pandas as pd
 ballots = pd.DataFrame(columns=['Allie', 'Billy', 'Candace'],
                        data=[*2*[[5,      4,       0]],
```

Comment:
Could the stray * also be removed?

Reply:
Oh, it’s not stray markdown but sequence unpacking operator and multiplication operator? 🤦🏽

Reply:
Yep :)